Makefile

```diff
@@ -0,0 +1,18 @@
+ARCHS=linux linux64 mac64 win32
+MANIFEST=manifest.json
+
+VERSION=`grep -Po '"'"version"'"\s*:\s*"\K([^"]*)' $(MANIFEST)`
```

Comment:
It looks like `-Po` is not available for grep on OSX by default.
Either:
- use something that would work on OX as well. I am very bad with scripts, but following seems to work on my machine:
  `grep -Eo '"version"\:\s*"[^"]+' manifest.json | grep -Eo '\d.*'`
- force OSX users to update grep for instance via
  `brew install grep --with-default-names`
  (and document it accordingly, the best would be to log something if VERSION is empty at this step)

Reply:
Oh, nice.  the -P option must be a GNU grep specific option, whereas the default grep on MacOSX is BSD one?
I will take the first approach in your suggestions.

Reply:
Hmm \d doesn't work for me.   I will use -Eo '[0-9].*' instead.  I hope it works on MacOSX.